litmus-portal/graphql-server/graph/schema.graphqls

```diff
@@ -4,465 +4,431 @@
-
-input ChaosWorkFlowInput {
-  workflow_id: String
-  workflow_manifest: String!
-  cronSyntax: String!
-  workflow_name: String!
-  workflow_description: String!
-  weightages: [WeightagesInput!]!
-  isCustomWorkflow: Boolean!
-  project_id: ID!
-  cluster_id: ID!
-}
-
-type ChaosWorkFlowResponse {
-  workflow_id: String!
-  cronSyntax: String!
-  workflow_name: String!
-  workflow_description: String!
-  isCustomWorkflow: Boolean!
-}
-
-input WorkflowRunInput {
-  workflow_id: ID!
-  workflow_run_id: ID!
-  workflow_name: String!
-  executed_by: String!
-  execution_data: String!
-  cluster_id: ClusterIdentity!
-  completed: Boolean!
-  isRemoved: Boolean
-}
-
-type PodLogResponse {
-  workflow_run_id: ID!
-  pod_name: String!
-  pod_type: String!
-  log: String!
-}
-
-input PodLog {
-  cluster_id: ClusterIdentity!
-  request_id: ID!
-  workflow_run_id: ID!
-  pod_name: String!
-  pod_type: String!
-  log: String!
-}
-
-input PodLogRequest {
-  cluster_id: ID!
-  workflow_run_id: ID!
-  pod_name: String!
-  pod_namespace: String!
-  pod_type: String!
-  exp_pod: String
-  runner_pod: String
-  chaos_namespace: String
-}
-
-type clusterRegResponse {
-  token: String!
-  cluster_id: String!
-  cluster_name: String!
-}
-
-type SSHKey {
-  publicKey: String!
-  privateKey: String!
-}
-
-input GitConfig {
-  ProjectID: String!
-  Branch: String!
-  RepoURL: String!
-  AuthType: AuthType!
-  Token: String
-  UserName: String
-  Password: String
-  SSHPrivateKey: String
-}
-type GitConfigResponse {
-  Enabled: Boolean!
-  ProjectID: String!
-  Branch: String
-  RepoURL: String
-  AuthType: AuthType
-  Token: String
-  UserName: String
-  Password: String
-  SSHPrivateKey: String
-}
-
-type ManifestTemplate {
-  template_id: ID!
-  manifest: String!
-  template_name: String!
-  template_description: String!
-  project_id: String!
-  project_name: String!
-  created_at: String!
-  is_removed: Boolean!
-  isCustomWorkflow: Boolean!
-}
-
-input TemplateInput {
-  manifest: String!
-  template_name: String!
-  template_description: String!
-  project_id: String!
-  isCustomWorkflow: Boolean!
-}
-
-type KubeObjectResponse {
-  cluster_id: ID!
-  kube_obj: String!
-}
-
-input KubeObjectData {
-  request_id: ID!
-  cluster_id: ClusterIdentity!
-  kube_obj: String!
-}
-
-input KubeObjectRequest {
-  cluster_id: ID!
-  object_type: String!
-  kube_obj_request: KubeGVRRequest!
-}
-
-input KubeGVRRequest {
-  group: String!
-  version: String!
-  resource: String!
-}
-
-type Query {
-  getWorkflowRuns(
-    workflowRunsInput: GetWorkflowRunsInput!
-  ): GetWorkflowsOutput! @authorized
-
-  getCluster(project_id: String!, cluster_type: String): [Cluster!]! @authorized
-
-  # Query to fetch manifest
-  getManifest(
-    projectID: String!
-    clusterID: String!
-    accessKey: String!
-  ): String! @authorized
-
-  # Query to fetch agent details based on project_id and agent_name
-  getAgentDetails(cluster_id: String!, projectID: String!): Cluster! @authorized
```

Comment:
This query has been removed, is it not being used anywhere?

Reply:
Have added back this query @Saranya-jena